src/libs/antares/args/include/antares/args/args_to_utf8.h

```diff
@@ -1,11 +1,14 @@
 #pragma once
 
+#include <utility>
+
 class IntoUTF8ArgsTranslator
 {
 public:
-	IntoUTF8ArgsTranslator(int argc, char**& argv);
+	IntoUTF8ArgsTranslator(int argc, char** argv);
+    std::pair<int, char**> convert();
 	~IntoUTF8ArgsTranslator();
 private:
-	int argc_ = 0;
-	char**& argv_;
+	int argc_;
```

Comment:
I guess that, because these **private** member variables are set by the constructor (and because this constructor is the only constructor the class has), we don't need to give them a default value in the current class declaration, right ?

Reply:
The default is indeed no longer given